ceph-salt-formula/salt/ceph-salt/ceph-osd.sls

```diff
@@ -11,7 +11,7 @@ deploy ceph osds ({{ loop.index }}/{{ dg_list | length }}):
   cmd.run:
     - name: |
         ceph orch device ls --refresh
-        echo '{{ dg_spec }}' | ceph orch osd create -i -
+        echo '{{ dg_spec }}' | ceph orch apply osd -i -
```

Comment:
I'm afraid we need to adapt the `dg_spec` too:
```
node1:~ # echo ''{"testing_dg_node1": {"host_pattern": "node1*", "data_devices": {"all": true}}}'' | ceph orch apply osd -i -
Error ENOENT: Spec needs a "service_type" key.
```

Reply:
hm, `service_type` does not seem to be documented at https://docs.ceph.com/docs/master/cephadm/drivegroups/#drivegroups

@sebastian-philipp ?

Reply:
@smithfarm I think this is the new format https://github.com/ceph/ceph/pull/33922/files#diff-74a24a0705ac114d3ec9a1bbb6f427deR135

Reply:
@ricardoasmarques OK, I implemented the new syntax at https://github.com/SUSE/sesdev/pull/178 but the deployment smoke test is still not green :-( Please take a look at the log there.